src/autoscaler/db/sqldb/lock_sqldb.go

```diff
@@ -158,7 +158,7 @@ func (ldb *LockSQLDB) Lock(lock *models.Lock) (bool, error) {
 				return err
 			}
 			if lastUpdatedTimestamp.Add(time.Second * time.Duration(fetchedLock.Ttl)).Before(currentTimestamp) {
-				ldb.logger.Info("lock-expired", lager.Data{"Owner": fetchedLock.Owner})
+				ldb.logger.Debug("lock-expired", lager.Data{"Owner": fetchedLock.Owner})
```

Comment:
This is an important and non-regular condition to check if the lock is expired without renew. So lets keep this as info only.